tests/test_pymafx_pipeline.py

```diff
@@ -0,0 +1,114 @@
+import argparse
+
+import mmcv
+import torch
+from demo.pymafx_estimate_smplx import prepare_data_with_pifpaf_detection
+from openpifpaf import decoder as ppdecoder
+from openpifpaf import network as ppnetwork
+from openpifpaf.predictor import Predictor
+from openpifpaf.stream import Stream
+from torch.utils.data import DataLoader
+
+from mmhuman3d.apis import init_model
+from mmhuman3d.data.datasets import build_dataset
+
+
+def _init_openpifpaf(parser):
```

Comment:
Shall we try to replace it with mmpose? If results are similar, we don't need to rely on openpifpaf.

Reply:
To align with the results of the `PyMAF-X`, I recommend keeping openpifpaf and changing it to mmpose in the next release.